monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -108,6 +108,10 @@ def _serialize_events(self, events: Sequence[AbstractAgentEvent]) -> JSONSeriali
 
         return serialized_events
 
+    def send_heartbeat(self, agent_id: AgentID, timestamp: float):
+        data = {"heartbeat_timestamp": timestamp}
```

Comment:
Since the endpoint is named "heartbeat", this can probably just be "timestamp". We can fix this in another PR, since it needs to be fixed in the endpoint as well.

Reply:
This'll get fixed by #2644.